tasks.py

```diff
@@ -23,101 +22,12 @@ def production(c):
     c.config.namespace = "hip-production"
 
 
-@invoke.task
-def build_ci_images(c):
-    """Build CircleCI test image using docker-compose"""
-    c.run("docker-compose -f docker-compose.yml -f docker-compose-test.yml build app")
-
-
-@invoke.task(
-    help={"command": "Passes a command to the container to run (ex: 'ls -la')"}
-)
-def run_in_ci_image(c, command):
-    """Runs command in the CircleCI test container.
-
-    Args:
-        command (str): A bash style command line command of variable length and composition.
-    Usage:
-        inv ci-run --command='pytest'
-    """
-    c.run(
-        f"docker-compose -f docker-compose.yml -f docker-compose-test.yml run --rm app sh -lc '{ command }'"
-    )
-
-
-@invoke.task
-def build_deployable(c):
```

Comment:
I notice this and the `build_deploy` tasks are still in the `tasks.py` file in jade-truffle. Do we need them here too?

Reply:
Can't sneak anything by you! I couldn't see a reason that we would need these on this project, but if you see value in them, I'd be happy to bring them back.

Reply:
I don't _think_ we need them here. I guess we can make a mental note about the fact that we removed them.